card.go

```diff
@@ -410,17 +410,30 @@ func fetchScryfallCardByFuzzyName(input string, isLang bool) (Card, error) {
 		}
 		if !isLang && card.Lang != "en" {
```

Comment:
Why doesn't this logic work for Uro/Aurochs?

Reply:
Let's check out [the fuzzy match here](https://api.scryfall.com/cards/named?fuzzy=Uro)

```
name: "Aurochs"
printed_name: "Uro"
lang: "es"
```

So we return "Oh, this card is Aurochs, right?" and then fuzzymatch on the actual name `Aurochs`.

Reply:
Oh I see, the logic is sort of backwards